pkg/products/marin3r/apiUsagePrometheusRules.go

```diff
@@ -80,10 +80,10 @@ func mapAlertsConfiguration(namespace, rateLimitUnit string, rateLimitRequestsPe
 					Alert: alertConfig.RuleName,
 					Annotations: map[string]string{
 						"message": fmt.Sprintf(
-							"RHOAM API usage is between %s and %s of the allowable threshold, %d requests per %s, during the last %s",
+							"Total API usage in your API Management service is between %s%% and %s%% percent of the allowable threshold, %d requests per %s, during the last %s",
```

Comment:
Is this correct syntax? `%s%%`? Should it not be `%s`?

Reply:
@pmccarthy `%s%%` would result in `90%`. The double % is the syntax required by fmr.Sprintf to produce a % symbol in the output. You can see that running here: https://play.golang.org/p/efuSxUz4vRl.
Although I see now I've got the word `percent` in there as well. I'll remove that.

Reply:
Ok cool, just said I'd ask in any case 👍